<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
<title>TestNG Report</title>
<style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}.invisible {display:none}</style>
</head>
<body>
<table>
<tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Retried</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr>
<tr><th colspan="7">FIAutomationReport</th></tr>
<tr><td><a href="#t0">WebAutomation</a></td><td class="num">0</td><td class="num attn">3</td><td class="num">0</td><td class="num">0</td><td class="num">150</td><td></td><td></td></tr>
</table>
<table id='summary'><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">FIAutomationReport</th></tr></tbody><tbody id="t0"><tr><th colspan="4">WebAutomation &#8212; failed (configuration methods)</th></tr><tr class="failedeven"><td rowspan="2">LoginTest</td><td><a href="#m0">afterMethod</a></td><td rowspan="1">1550596041820</td><td rowspan="1">8</td></tr><tr class="failedeven"><td><a href="#m1">beforeMethod</a></td><td rowspan="1">1550596041733</td><td rowspan="1">77</td></tr><tr><th colspan="4">WebAutomation &#8212; skipped</th></tr><tr class="skippedeven"><td rowspan="3">LoginTest</td><td><a href="#m2">checkCartItemList</a></td><td rowspan="1">1550596041833</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m3">initLogin</a></td><td rowspan="1">1550596041817</td><td rowspan="1">0</td></tr><tr class="skippedeven"><td><a href="#m4">shopItems</a></td><td rowspan="1">1550596041832</td><td rowspan="1">0</td></tr></tbody>
</table>
<h2>WebAutomation</h2><h3 id="m0">LoginTest#afterMethod</h3><table class="result"><tr class="param"><th>Parameter #1</th></tr><tr class="param stripe"><td>public void LoginTest.initLogin(java.lang.String,java.lang.String) throws java.lang.Exception</td></tr><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException
	at ReusableFunctions.captureScreenShot(ReusableFunctions.java:15)
	at TestBase.afterMethod(TestBase.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:669)
	at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:644)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:516)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:170)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:790)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:143)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at org.testng.TestRunner.privateRun(TestRunner.java:763)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:437)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m1">LoginTest#beforeMethod</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException
	at org.openqa.selenium.remote.service.DriverCommandExecutor.&lt;init&gt;(DriverCommandExecutor.java:63)
	at org.openqa.selenium.firefox.FirefoxDriver$FirefoxDriverCommandExecutor.&lt;init&gt;(FirefoxDriver.java:117)
	at org.openqa.selenium.firefox.FirefoxDriver.&lt;init&gt;(FirefoxDriver.java:162)
	at DriverFactory.FireFoxDriverManager.createDriver(FireFoxDriverManager.java:59)
	at DriverFactory.DriverManager.getDriver(DriverManager.java:25)
	at TestBase.beforeMethod(TestBase.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:669)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:496)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:170)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:790)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:143)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at org.testng.TestRunner.privateRun(TestRunner.java:763)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:437)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m2">LoginTest#checkCartItemList</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.Throwable: Method LoginTest.checkCartItemList()[pri:0, instance:LoginTest@1f7030a6] depends on not successfully finished methods
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:97)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at org.testng.TestRunner.privateRun(TestRunner.java:763)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:437)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m3">LoginTest#initLogin</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.NullPointerException
	at org.openqa.selenium.remote.service.DriverCommandExecutor.&lt;init&gt;(DriverCommandExecutor.java:63)
	at org.openqa.selenium.firefox.FirefoxDriver$FirefoxDriverCommandExecutor.&lt;init&gt;(FirefoxDriver.java:117)
	at org.openqa.selenium.firefox.FirefoxDriver.&lt;init&gt;(FirefoxDriver.java:162)
	at DriverFactory.FireFoxDriverManager.createDriver(FireFoxDriverManager.java:59)
	at DriverFactory.DriverManager.getDriver(DriverManager.java:25)
	at TestBase.beforeMethod(TestBase.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:131)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:669)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:496)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:170)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:790)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:143)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at org.testng.TestRunner.privateRun(TestRunner.java:763)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:437)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
<h3 id="m4">LoginTest#shopItems</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.Throwable: Method LoginTest.shopItems()[pri:0, instance:LoginTest@1f7030a6] depends on not successfully finished methods
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:97)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at org.testng.TestRunner.privateRun(TestRunner.java:763)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:398)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:41)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:437)
	at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:67)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p>
</body>
</html>
